usr/local/www/interfaces_bridge_edit.php

```diff
@@ -237,380 +271,295 @@
-	     "checking the full duplex link status.  This is the default for " .
-	     "interfaces added to the bridge."); ?></span>
-				 <p class="vexpl"><span class="red"><strong>
-				  <?=gettext("Note:"); ?><br />
-				  </strong></span>
-		 <?=gettext("The interfaces selected here will be removed from default autoedge status."); ?>
-					</p></td>
-			    </tr>
-                <tr style="display:none" id="sprtable9">
-                  <td valign="top" class="vncell"><?=gettext("Sticky ports"); ?></td>
-                  <td class="vtable">
-					<select name="static[]" class="formselect" multiple="multiple" size="3">
-						<?php
-							foreach ($ifacelist as $ifn => $ifdescr) {
-								echo "<option value=\"{$ifn}\"";
-								if (stristr($pconfig['static'], $ifn))
-									echo " selected=\"selected\"";
-								echo ">{$ifdescr}</option>";
-							}
-						?>
-					</select>
-                    <br />
-                    <span class="vexpl">
-	     <?=gettext("Mark an interface as a \"sticky\" interface.  Dynamically learned " .
-	     "address entries are treated as static once entered into the " .
-	     "cache.  Sticky entries are never aged out of the cache or " .
-	     "replaced, even if the address is seen on a different interface."); ?>
-					</span></td>
-			    </tr>
-                <tr style="display:none" id="sprtable10">
-                  <td valign="top" class="vncell"><?=gettext("Private ports"); ?></td>
-                  <td class="vtable">
-					<select name="private[]" class="formselect" multiple="multiple" size="3">
-						<?php
-							foreach ($ifacelist as $ifn => $ifdescr) {
-								echo "<option value=\"{$ifn}\"";
-								if (stristr($pconfig['private'], $ifn))
-									echo " selected=\"selected\"";
-								echo ">{$ifdescr}</option>";
-							}
-						?>
-					</select>
-                    <br />
-                    <span class="vexpl">
-	     <?=gettext("Mark an interface as a \"private\" interface.  A private interface does not forward any traffic to any other port that is also " .
-	     "a private interface."); ?>
-					</span></td>
-			    </tr>
-                <tr>
-                  <td width="22%" valign="top">&nbsp;</td>
-                  <td width="78%">
-		    <input type="hidden" name="bridgeif" value="<?=htmlspecialchars($pconfig['bridgeif']); ?>" />
-                    <input name="Submit" type="submit" class="formbtn" value="<?=gettext("Save"); ?>" />
-                    <input type="button" class="formbtn" value="<?=gettext("Cancel");?>" onclick="window.location.href='<?=$referer;?>'" />
-                    <?php if (isset($id) && $a_bridges[$id]): ?>
-                    <input name="id" type="hidden" value="<?=htmlspecialchars($id);?>" />
-                    <?php endif; ?>
-                  </td>
-                </tr>
-              </table>
-</form>
-<?php include("fend.inc"); ?>
-</body>
-</html>
+
+$section->addInput(new Form_StaticText(
```

Comment:
I think you should use $section->setHelp here; it would provide the same functionality